eiffel-vocabulary/EiffelTestSuiteStartedEvent.md

```diff
@@ -85,6 +85,12 @@ __Legal targets:__ [EiffelActivityTriggeredEvent](../eiffel-vocabulary/EiffelAct
 __Multiple allowed:__ No  
 __Description:__ Identifies the activity or test suite of which this event constitutes a part.
 
+### PRECURSOR
+__Required:__ No  
+__Legal targets:__ [EiffelTestSuiteStartedEvent](./EiffelTestSuiteStartedEvent.md)  
+__Multiple allowed:__ Yes  
+__Description:__ Used to declare temporal relationships between test suite activities in a pipeline step, i.e. what other test suite (or or test suites) preceded this activity. This link type applies primarily to non event-triggered test suite executions.
```

Comment:
Yes. If I would link to *activity* it could confuse the reader that this is a sub-type or something of an activity event type. The same for the pipeline link, as this text states pipeline step and not just pipeline. Do you have a good suggestion on what links to put in there?

Reply:
I've updated the PR now to also allow PRECURSOR links between TCT events, and thus rewritten some parts of the PR that relate to that update.